index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio | Abenezer Fekadu</title>
+  <link rel="stylesheet" href="./style.css">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="wrapper">
+    
+    
+      <nav class="navbar">
+
+        <header class="navbar-brand"><a href="home-page-link">Abenezer</a></header>
+    
+      
+        <ul class="navbar-list">
+          <li class="navbar-item">Portofolio</li>
+          <li class="navbar-item">About</li>
+          <li class="navbar-item">Contact</li>
+          <li class="navbar-item"><img src="./img/666162.png" alt="" class="nav-icon"></li>
+        </ul>
+      <div class="navbar-toggle">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+      </nav>
+      <section class="header">
+      <div class="header-text"><h1>Hey There. <span>I'm Abenezer</span> </h1>
+      </div>
```

Comment:
The goal of this project is to code the given design pixel perfectly like maybe you have seen on some job ads. Sı please stick to the Figma design as the previous reviewer requested and fix the font-size and font-weight like on the below.
<img width="637" alt="Screenshot 2023-01-19 at 7 29 09 PM" src="https://user-images.githubusercontent.com/25887253/213500055-1c8b2fe6-bed0-44d4-a64f-56c67f3ae042.png">
<img width="974" alt="Screenshot 2023-01-19 at 7 30 16 PM" src="https://user-images.githubusercontent.com/25887253/213500812-35214ccd-064d-40b0-a785-add9a546b811.png">